src/style.css

```diff
@@ -1,3 +1,65 @@
+:root {
+  --nav-color: #00acdc;
+}
+
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav {
+  line-height: 36px;
+  text-transform: uppercase;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  gap: 20px;
+  font-size: 12px;
+  font-weight: 500;
+  text-align: center;
+}
+
+.nav-link {
+  color: black;
+  text-decoration: none;
+}
```

Comment:
At the moment you have one main problem, you should set the height for `nav__link` as it is wrote in the requirements to the task. As you can see at tests, every nav link has 100% of the header height.
But your links have only their line height
![image](https://github.com/mate-academy/layout_moyo-header/assets/64101898/a795df4f-690c-431e-a670-54fa5598d03f)